doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| HA interface                                            | HA interface API that allows Halon to control Motr and allows Motr to receive cluster state information from Halon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| In-storage Compute Subsystem                            | Another name for the function-shipping subsystem.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Then, what is function-shipping.

Please go to function-shipping document, and find its description.